cmake/PackageProject.cmake

```diff
@@ -157,7 +157,9 @@ function(adobe_contract_checking_package_project)
   # download ForwardArguments
   FetchContent_Declare(
     _fargs
-    URL https://github.com/polysquare/cmake-forward-arguments/archive/8c50d1f956172edb34e95efa52a2d5cb1f686ed2.zip)
+    URL https://github.com/polysquare/cmake-forward-arguments/archive/8c50d1f956172edb34e95efa52a2d5cb1f686ed2.zip
+    DOWNLOAD_EXTRACT_TIMESTAMP "ON"
```

Comment:
I think it would probably be better to do this by setting [the policy](https://cmake.org/cmake/help/latest/policy/CMP0135.html) to `NEW` than by passing the argument everywhere.  WDYT?

Reply:
Its still generating warnings about FetchContent_Populate being deprecated and the suggested replacement doesn't work. Leaving for now (template is too complicated!).

Reply:
The `FetchContent_Populate` warning has nothing to do with whether we use this parameter or set the policy.  Totally separate issue.  `FetchContent_Populate` is invoked from elsewhere.